frontend/docker/Config.js

```diff
@@ -8,6 +8,7 @@ export const Config = {
   'sessionRefreshInterval': 10 * 60 * 1000,
   'languages': {
     'en': 'English',
+    'da': 'Danish',
```

Comment:
I think that should be 'Dansk'? So far, we've been using the language's name in the language itself